recipes/qutip/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+${PYTHON} setup.py install
```

Comment:
Sadly, there is a bug which means you do currently need the .bat file, but that needs fixing in conda-build.

> I mean when install qutip normally (linux, win, mac) I just run

When the the software isn't being re-packaged, that is fine. For cases where it is (i.e. conda-build) then we need to be more careful. One reference: https://github.com/conda/conda/issues/508. We really need to write this up (and give the full reasons why it is a good idea).